manager/src/main/java/com/redhat/service/bridge/manager/workers/id/WorkerIdProvider.java

```diff
@@ -4,13 +4,33 @@
 
 import javax.enterprise.inject.Produces;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class WorkerIdProvider {
 
-    private static final String WORKER_ID = UUID.randomUUID().toString();
+    private static final String RANDOM_STATIC_WORKER_ID = UUID.randomUUID().toString();
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WorkerIdProvider.class);
+
+    private String findPodNameFromKubernetes() {
+        String hostname = System.getenv("HOSTNAME");
+        LOGGER.trace("POD id from hostname: {}", hostname);
+
+        if (hostname != null && !hostname.isEmpty() && !"localhost".equals(hostname)) {
```

Comment:
There does not appear to be any guarantee that the environment variable `HOSTNAME` returns a lower case `String`. For the sake of avoiding future potential issues where someone has it set as `LOCALHOST` etc it'd be advisable to use `equalsIgnoreCase(..)`.

I don't suppose you want any unit tests for the change you've made too?

Reply:
sounds as good suggestion